backend/backend/users/api/serializers.py

```diff
@@ -35,20 +37,85 @@ class ProfileSerializer(TaggitSerializer, serializers.ModelSerializer):
 
     tags = ConstellateTagListSerializerField()
```

Comment:
This is worth thinking over - can we just send the list of strings instead of the objects?